lib/rules.js

```diff
@@ -65,7 +65,7 @@ function importPlugin(pluginRules, pluginName) {
  * @param {string} ruleId Rule id (file name).
  * @returns {Function} Rule handler.
  */
-function get(ruleId) {
+function getHandler(ruleId) {
```

Comment:
Function renamed to avoid Node v4.x.x bug that treats shorthand properties named get as syntax errors.